src/node/eviction.cpp

```diff
@@ -175,14 +176,24 @@ void ProtectEvictionCandidatesByRatio(std::vector<NodeEvictionCandidate>& evicti
     EraseLastKElements(eviction_candidates, ReverseCompareNodeTimeConnected, remaining_to_protect);
 }
 
-[[nodiscard]] std::optional<NodeId> SelectNodeToEvict(std::vector<NodeEvictionCandidate>&& vEvictionCandidates)
+[[nodiscard]] std::optional<NodeId> SelectNodeToEvict(std::vector<NodeEvictionCandidate>&& vEvictionCandidates, bool force)
 {
     // Protect connections with certain characteristics
 
     ProtectNoBanConnections(vEvictionCandidates);
 
     ProtectOutboundConnections(vEvictionCandidates);
 
+    // No inbound or unprotected connections left, return early
+    if (vEvictionCandidates.empty()) return std::nullopt;
```

Comment:
c5e7e6563201965e88e07aa8da8f26e17fc1b4db

"unprotected" here is sloppy. I'd just drop the comment.

Reply:
ok